<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMarkDialog</class>
 <widget class="QDialog" name="NewMarkDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="windowTitle">
   <string>Add mark</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <item>
    <widget class="QWidget" name="leftWidget" native="true">
     <layout class="QGridLayout" name="leftLayout">
      <item row="3" column="2">
       <widget class="QLineEdit" name="absoluteLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="modernRadioButton">
        <property name="text">
         <string>Modern</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="markLineEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="dateLabel">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="absoluteCheckBox">
        <property name="text">
         <string>Absolute</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QDateEdit" name="dateEdit">
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="markLabel">
        <property name="text">
         <string>Mark</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QRadioButton" name="classicRadioButton">
        <property name="text">
         <string>Classic</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightLayout">
     <item>
      <widget class="QLabel" name="commentLabel">
       <property name="text">
        <string>Comment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="commentPlainTextEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dateEdit</tabstop>
  <tabstop>markLineEdit</tabstop>
  <tabstop>classicRadioButton</tabstop>
  <tabstop>modernRadioButton</tabstop>
  <tabstop>absoluteCheckBox</tabstop>
  <tabstop>absoluteLineEdit</tabstop>
  <tabstop>commentPlainTextEdit</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>absoluteCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>absoluteLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>NewMarkDialog</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>check()</slot>
 </slots>
</ui>
